About/Publications/PublishedWorks.md

```diff
@@ -0,0 +1,2 @@
+Signed-off-by: Jon Lawrence <jon@mpgstudios.com>
```

Comment:
Please remove the sign-off from the file. DCO sign-offs need to be part of the commit message.